docs/SERVER_SETUP.md

```diff
@@ -65,7 +65,7 @@ You can check your Java version from the shell/command line using: ``java -versi
 * Install the "temporal tables" extension for Postgres (<https://github.com/arkhipov/temporal_tables>)
   * **Windows**: Download the latest version for your Postgres version: <https://github.com/arkhipov/temporal_tables/releases/latest>, then copy the DLL from the project into the PostgreSQL's lib directory and the .sql and .control files into the directory share\extension.
   * **Linux** (see <https://github.com/arkhipov/temporal_tables#installation)>
-
+* Add the PostgreSQL path (/etc/PostgreSQL/10/bin) to Environment Variables
```

Comment:
Why is this line added here? The script belongs to the temporal tables topic

Reply:
where should it be added?

Reply:
Just after the script path I would say

Reply:
the script path - what is that ?

Reply:
The script block (line 69ff) I meant but did not type.